pkg/config/db.go

```diff
@@ -116,6 +113,33 @@ type (
 		Tables []*Table `yaml:"tables" json:"tables"`
 	}
 
+	ListenerV2 struct {
+		ProtocolType  string            `yaml:"protocol_type" json:"protocol_type"`
+		SocketAddress *SocketAddress    `yaml:"socket_address" json:"socket_address"`
+		Config        *ListenerV2Config `yaml:"config" json:"config"`
+		Executor      string            `yaml:"executor" json:"executor"`
+	}
+
+	ListenerV2Config struct {
+		ServerVersion string `yaml:"server_version" json:"server_version"`
+		Users         *Users `yaml:"users" json:"users"`
+	}
+
+	Users struct {
+		Username string `yaml:"username" json:"username"`
+		Password string `yaml:"password" json:"password"`
+	}
```

Comment:
Emm... I think there should be 2 possible patterns:
1. `Users` should be `[]*User`, the struct of `User` is like current `Users`.
2. Or, the `Users` should be `map[string]string`, key means  username and value means password.
@dk-lockdown @AlexStocks @luky116 What do you guys think?

Reply:
Get, i think []*User is better.